data/trie/trieStorageManager.go

```diff
@@ -408,14 +411,19 @@ func (tsm *trieStorageManager) takeSnapshot(snapshot *snapshotsQueueEntry, msh m
 		return
 	}
 
-	maxTrieLevelInMemory := uint(5)
-	err = newRoot.commit(true, 0, maxTrieLevelInMemory, tsm.db, db)
-	if err != nil {
-		log.Error("trie storage manager: commit", "error", err.Error())
+	if snapshot.newDb {
```

Comment:
maybe use a StopWatch to measure the time taken by this method?

Reply:
Used a stopWatch in the accountsDB to measure the time it takes to snapshot/checkpoint the whole state.